joern-cli/frontends/pysrc2cpg/src/test/scala/io/joern/pysrc2cpg/cpg/MemberCpgTests.scala

```diff
@@ -0,0 +1,25 @@
+package io.joern.pysrc2cpg.cpg
+
+import io.joern.pysrc2cpg.Py2CpgTestContext
+import org.scalatest.freespec.AnyFreeSpec
+import org.scalatest.matchers.should.Matchers
+import io.shiftleft.semanticcpg.language._
+
+class MemberCpgTests extends AnyFreeSpec with Matchers {
+
+  lazy val cpg = Py2CpgTestContext.buildCpg("""
+      |class Foo():
+      |   x = 'foo'
```

Comment:
Could we also add a test for 


```python
class Foo():

  def __init__(self, x):
    self.x = x
```
Where `x` would be some member here?

Reply:
Or is this out of scope?

Reply:
Yeah, good point.

Reply:
Deferring to follow-up PR.